src/base/ui_object.js

```diff
@@ -188,14 +188,33 @@ export default class UIObject extends BaseObject {
   /**
    * ensures the creation of this ui component
    * @method _ensureElement
+   * @param {Object} options the options object
    * @private
    */
-  _ensureElement() {
+  _ensureElement(options) {
```

Comment:
I'd suggest not doing this in `ensureElement` directly, as it's used by every component/plugin that renders an element. Instead, since it's a feature specific to the playback implementations, I'd move this logic to either the `Playback` base class or the HTML5 video playback.